stable/anchore-engine/templates/anchore_upgrade_role.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.anchoreGlobal.usePreupgradeHook -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "anchore-engine.fullname" . }}-upgrade-role
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "anchore-engine.fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    {{- with .Values.anchoreGlobal.labels }}
+    {{ toYaml . | nindent 4 }}
+    {{- end }}
+  {{- if .Values.anchoreGlobal.annotations }}
```

Comment:
I think all these templates should probably have the labels/annotations/etc from  values.[service]upgradeJob as well as globals.

Reply:
added